ui/user_settings.js

```diff
@@ -362,6 +362,7 @@ function add_blockchain_account(event) {
             if (blockchain == 'ETH') {
                 recreate_ethchain_per_account_table(result['per_account']['ETH']);
             } else if (blockchain == 'BTC') {
+                create_blockchain_balances_tables(result)
```

Comment:
I don't think this is correct here. Why did you add it?

Reply:
The table for displaying the BTC balances was not showing up, and a null reference error was thrown in console when the BTC address was added.

Looking through the code for the `create_blockchain_balances_tables`, I saw that it created the element, and handler for the BTC balances table.

Placing it here fixed the issue. Do you have a better suggestion?